### PR TITLE
build: dockerization

### PR DESCRIPTION
**Pull request by inchori**

### CODE REVIEW

**Comment by inchori** [comment]
When the #4 is merged, I'll re-open this PR.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Install minimum necessary dependencies,
+ENV PACKAGES make git gcc
+RUN apt-get update -y
+RUN apt-get install -y $PACKAGES
+
+COPY . /src/panacea-dep-consumer
+
+WORKDIR /src/panacea-dep-consumer
+
+RUN make clean && make build
```
Comment (gyuguen) [inline Dockerfile]:
```suggestion
RUN make clean build
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+FROM debian:bullseye-slim
+
+COPY --from=build-env /src/panacea-dep-consumer/build/consumerd /usr/bin/consumerd
+
+RUN chmod +x /usr/bin/consumerd
+
+CMD ["/consumerd"]
+
+EXPOSE 8080
```
Comment (gyuguen) [inline Dockerfile]: Can you need this expose option?
I know that the default port for this service is 8080, which is not use.

**Review by gyuguen — COMMENTED** [review]
LGTM

File: Dockerfile (hunk 2 above)
Comment (inchori) [reply Dockerfile]: Yes, the user can set listen address address with port. This option is not necessary and without this option the works well. Fixed it in [970f962](https://github.com/medibloc/panacea-dep-consumer/pull/5/commits/970f9627a64c2a9947b3ba2c3812859f6b0a4cff).

**Review by gyuguen — APPROVED** [review]
LGTM

**Review by youngjoon-lee — APPROVED** [review]
lgtm!

**Review by audtlr24 — APPROVED** [review]
LGTM!

**Pull request merged**